packages/main/src/BusyIndicator.js

```diff
@@ -78,6 +79,27 @@ const metadata = {
 		active: {
 			type: Boolean,
 		},
+
+		/**
+		 * Defines the delay in milliseconds, after which the busy indicator will be visible on the screen.
+		 *
+		 * @type {Integer}
+		 * @defaultValue 1000
+		 * @public
+		 */
+		delay: {
+			type: Integer,
+			defaultValue: 1000,
+		},
+
+		/**
+		 * Defines if the control is currently in busy state.
```

Comment:
"control" is not appropriate term here. Replace it with "component"

Reply:
Done.